criu/config.c

```diff
@@ -697,6 +698,7 @@ int parse_options(int argc, char **argv, bool *usage_error, bool *has_exec_cmd,
 		{ "file-validation", required_argument, 0, 1098 },
 		{ "lsm-mount-context", required_argument, 0, 1099 },
 		{ "network-lock", required_argument, 0, 1100 },
+		BOOL_OPT("mounts-v2", &opts.mounts_v2),
```

Comment:
I think mounts_v2 should be used by default if it is supported. We can add --force_mount_v1 to disable mounts_v2.

Reply:
That's a good point, Thanks!

We have "--mounts-compat" option in vzcriu, and we also have a fallback if mount-v2 is not supported to switch to compat mode. Does this name sounds ok for you?

Reply:
`--mounts-compat` sounds good

Reply:
I already prepared a rework with `--force-mounts-compat`, I believe using the word `force` enlightens the fact that we also have an implicit fallback to compat mode if needed kernel features are absent.

Reply:
Updated to --mntns-compat-mode as discussed with @avagin